src/tests/ftest/control/dmg_server_set_logmasks.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2022 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+
+
+from control_test_base import ControlTestBase
+
+class DmgServerSetLogmasksTest(ControlTestBase):
+    # pylint: disable=too-many-ancestors
+    """Test Class Description:
+    Verify the server set-logmasks function of the dmg tool.
+    :avocado: recursive
+    """
+
+    def test_dmg_server_set_logmasks(self):
```

Comment:
Could it be possible to test that the mask has been  effectively set ?

Reply:
As discussed with @mjmac this test can be extended in future to perform validation of the engine logmasks change but the initial scope is just to validate the command returns successfully to prevent against regressions related to the RPC flow.